src/runtime-rs/crates/hypervisor/src/device/driver/vfio.rs

```diff
@@ -456,7 +457,13 @@ impl Device for VfioDevice {
             .await
             .context("failed to increase attach count")?
         {
-            return Err(anyhow!("attach count increased failed as some reason."));
+            warn!(
```

Comment:
Could we detect the case that the device is attached more than once due to multiple containers?
Right now this warn doesn't tell users why multiple time attachment happen and this seems like a bug warn to users.

Reply:
First of all, we need ensure the device can be attached only once, right ? No matter which case is.
And, the warning is appropriate for such cases,  multi container or other cases trying to attach devices more than once error!